src/persist/fileSystem.ts

```diff
@@ -1,4 +1,3 @@
-import { Buffer } from 'buffer';
```

Comment:
I just realize this line is not needed.

I ran the following command `yarn run build` with `"noUnusedLocals": true` in tsconfig and it returns the following error.
```
src/persist/fileSystem.ts:1:1 - error TS6133: 'Buffer' is declared but its value is never read.
```